Automattic/jetpack
# Use webpack config JS package in Jetpack

#### Changes proposed in this Pull Request:

This shouldn't change anything visible in this iteration, but double-checking
might be a good idea.
    
This does take the opportunity to fix the extension build to keep
plugins and blocks in the order defined in extensions/index.json,
where #20994 had put all the blocks before all the plugins.

This also replaces the uses of `@automattic/calypso-build`'s jest configuration with local config.

#### Jetpack product discussion


PT: p9dueE-3MG-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Does it build?
* Is CI happy?
* See that things touched here still work right.
